runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5278,12 +5285,21 @@ class INTERPRETER_CLASS
 		recordJNIReturn(REGISTER_ARGS, bp);
 		restoreSpecialStackFrameLeavingArgs(REGISTER_ARGS, bp);
 
-#if JAVA_SPEC_VERSION >= 20
+#if JAVA_SPEC_VERSION >= 21
 		/* Set the execution state after the downcall as required in the linker options. */
 		if (NULL != returnState) {
-			*returnState = (I_32)errno;
+		/* The error code layout on Windows in JDK21 is changed to a segment for an array of integers
+		 * which saves the return value of GetLastError(), WSAGetLastError() and errno.
+		 */
```

Comment:
Please correct the indentation of the body of this `if` statement.

Reply:
Fixed the indentation around the changes in there.